docs/modules/development/pages/rest/snmp_configuration.adoc

```diff
@@ -168,76 +168,89 @@ It also shows the version limitations, default values, and the restrictions - if
 | readCommunity           | read-community            | public       | v1, v2c               |
 | writeCommunity          | write-community           | private      | v1, v2c               |
 | securityName            | security-name             | opennmsUser  | v3                    |
-| securityLevel           | security-level            | noAuthNoPriv | v3                    | Integer value, which can be null, 1, 2, or 3. <ul><li>1 means noAuthNoPriv</li><li>2 means authNoPriv</li><li>3 means authPriv</li></ul> If you do not set the security level manually it is determined automatically: <ul><li>if no authPassPhrase set the securityLevel is 1</li><li>if a authPassPhrase and no privPassPhrase is set the security level is 2.</li><li>if a authPassPhrase and a privPassPhrase is set the security level is 3.</li></ul>
+| securityLevel           | security-level            | noAuthNoPriv | v3                    | Integer value, which can be `null`, `1`, `2`, or `3`.
+See note below table for explanation of these values.
 | authPassPhrase          | auth-passphrase           | 0p3nNMSv3    | v3                    |
-| authProtocol            | auth-protocol             | MD5          | v3                    | only MD5, SHA, SHA-224, SHA-256, SHA-512 are valid arguments
+| authProtocol            | auth-protocol             | MD5          | v3                    | only `MD5`, `SHA`, `SHA-224`, `SHA-256`, `SHA-512` are valid arguments
 | privPassPhrase          | privacy-passphrase        | 0p3nNMSv3    | v3                    |
-| privProtocol            | privacy-protocol          | DES          | v3                    | only DES, AES, AES192 or AES256 are valid arguments.
+| privProtocol            | privacy-protocol          | DES          | v3                    | only `DES`, `AES`, `AES192`, or `AES256` are valid arguments.
 | engineId                | engine-id                 |              | v3                    |
 | contextEngineId         | context-engine-id         |              | v3                    |
 | contextName             | context-name              |              | v3                    |
 | enterpriseId            | enterprise-id             |              | v3                    |
 |===
 
+For the `securityLevel` field:
+
+* 1 means `noAuthNoPriv`.
+* 2 means `authNoPriv`.
+* 3 means `authPriv`.
+
+If you do not set the security level manually it is determined automatically:
+
+* If no `authPassPhrase` set, the securityLevel is 1.
```

Comment:
```suggestion
* If no `authPassPhrase` set, the security level is 1.
```

Reply:
I was not clear if this was an actual field or the concept of security level. Whichever one it is, it should be consistent in this file and the snmp_configuration.adoc file.

Reply:
should be one word to reference the implied field value.